1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,49 @@
 // Write your code here
+let stop = document.getElementById("btnStop");
+let back = document.getElementById("btnBack");
+let forward = document.getElementById("btnForward");
+let autoBack = document.getElementById("btnAutoBack");
+let autoForward = document.getElementById("btnAutoForward");
+let img = document.querySelector("img");
+const dogImages = [
+  "images/dog1.png",
+  "images/dog2.png",
+  "images/dog3.png",
+  "images/dog4.png",
+];
+let currentImageIndex = 0;
+
+forward.addEventListener("click", () => {
+  currentImageIndex++; // This is incrementing the index of the dogImages array by 1
+  currentImageIndex = currentImageIndex % dogImages.length;
+  img.src = dogImages[currentImageIndex];
+  console.log(img.src);
+  console.log(currentImageIndex);
+});
+
+back.addEventListener("click", () => {
```

Comment:
Instead of creating all the functionality in the EventListener, we should create it as its own function, then call the function within the eventlistener. That's because we want to reuse the forward and back when we get to the autoforward and autoback.